src/styles/index.scss

```diff
@@ -1,3 +1,255 @@
+$stars-size: 16px;
+
 body {
   margin: 0;
+  font-family: Roboto, 'Times New Roman', serif;
+  font-weight: 500;
+}
+
+.container {
+  --column-count: 4;
+
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  justify-content: center;
+  gap: 46px 48px;
+}
+
+@media (min-width: 488px) {
+  .container {
+    --column-count: 2;
+  }
+}
+
+@media (min-width: 768px) {
+  .container {
+    --column-count: 3;
+  }
+}
+
+@media (min-width: 1024px) {
+  .container {
+    --column-count: 4;
+  }
+}
+
+/* ====== Header ====== */
+.header {
+  display: flex;
+  height: 60px;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  padding: 3px 0 0;
+}
+
+.menu {
+  font-size: 12px;
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.link {
+  position: relative;
+  margin: 0 10px;
+  height: 60px;
+  align-items: center;
+  text-decoration: none;
+  color: #060b35;
+  display: flex;
+}
+
+.menu li:first-child a {
+  margin-left: 0;
+}
+
+.menu li:last-child a {
+  margin-right: 0;
+}
+
+.link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+  margin: 0 10px;
+  height: 60px;
+  align-items: center;
+  text-decoration: none;
+  display: flex;
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
+  border-radius: 2px;
+  transform-origin: bottom left;
+}
```

Comment:
The `::after` pseudo-element used in `.is-active` class should be consistent with the one in `.link:hover` (lines 104-106). Both are used to modify the link's appearance on hover and active states, so they should likely share the same styles for consistency.